telemetry/definitions/commonDefinitions.json

```diff
@@ -3556,6 +3566,14 @@
                     "type": "codeTransformSessionId",
                     "required": true
                 },
+                {
+                    "type": "codeTransformRuntimeError",
+                    "required": true
```

Comment:
Should be `false` because it's not always applicable. Either way this was merged recently in my PR.

Reply:
Noted: should make sure this is updated to false during merge/rebase